app/helpers/bootstrap_icons_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Helper to render Bootstrap Icons
+module BootstrapIconsHelper
+  # refer https://icons.getbootstrap.com/
+  def icon(icon, options = {})
```

Comment:
I think I'd move this method to the existing `ApplicationHelper` and maybe name the method `bootstrap_icon` for clarity. Maybe add a simple test too?